.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,76 @@
+on:
+  pull_request:
+    paths:
+      - .github/workflows/unit-tests.yml
+      - "**/*.go"
+      - "chain/**"
+      - "cmd/**"
+      - "dot/**"
+      - "internal/**"
+      - "lib/**"
+      - "pkg/**"
+      - "tests/**"
+      - go.mod
+      - go.sum
+      - Makefile
+name: integration-tests
+env:
+  GO111MODULE: on
+
+jobs:
+  unit-tests:
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        go-version: [1.17.x]
+        platform: [ubuntu-latest]
+        packages: [
+          github.com/ChainSafe/gossamer/dot/rpc/modules
+        ]
```

Comment:
I don't think there is much point in having a matrix, what do you think? Just run the integration tests with one single `go test -tags integration ./...` command from the root?

Reply:
In previous retrospectives, I remember we talked about using the Github actions matrix feature to parallelize tests as some of them take a lot of time and others just go quickly. So I don't think it's all bad to have the packages that contain integration tests being added there.

Reply:
Fair enough, although remove the platform and go-version. We will re-add them once we need them to contain multiple values. So far we never did.